RELEASE.md

```diff
@@ -1,5 +1,10 @@
 # RELEASE
 
 - Merge all PRs intended for inclusion in the new release.
+- Create a new branch for the release.
+- Run `npm run bundle-all` to bundle the actions into JavaScript files.
+- Commit the changes.
+- Open a pull request for the release branch, requesting approval from relevant teams.
 - Run `git tag -s "<TAG>" -m "<TAG>" && git push origin "<TAG>"`.
+- Merge the pull request.
```

Comment:
This appears to be out of order, as it's going to happen after the tag is made.

This should probably include a `git checkout main && git pull` after the PR is merged, to ensure that the local `main` branch is up to date before making the tag.

If you want to get even fancier, you can steal [this](https://github.com/fastly/cli/blob/d5e6d6724f75fef9a264ff14839657c4e4eb5766/RELEASE.md?plain=1#L11) which will for creating the tag even if the name of the remote is not `origin` (which is often the case for me).